### PR TITLE
Updated the FCI spectral responses as of April 2022

### PR DESCRIPTION
**Pull request by adybbroe**
Signed-off-by: Adam.Dybbroe <a000680@c21856.ad.smhi.se>

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [x] Tests passed: Passes ``pytest pyspectral`` 
 - [x] Passes ``flake8 pyspectral`` 
 - [x] Fully documented

### CODE REVIEW

File: rsr_convert_scripts/fci_rsr.py [hunk 1]
```diff
@@ -22,63 +22,52 @@
...
-        self.rsr = {'wavelength': wavelength[::-1], 'response': response[::-1]}
+        bandnames = ncf.variables['channel_id'][:]
+        for idx, band_name in enumerate(bandnames):
+            if band_name == self.bandname:
+                self.rsr = {'wavelength': wvl[:, idx], 'response': resp[:, idx]}
 
 
 def main():
-    """Main function creating the internal Pyspectral hdf5 output for FCI."""
+    """Ccreate the internal Pyspectral hdf5 output for FCI."""
```
Comment (sfinkens) [inline rsr_convert_scripts/fci_rsr.py:70]: Typo: Ccreate

File: rsr_convert_scripts/fci_rsr.py [hunk 2]
```diff
@@ -22,63 +22,52 @@
...
-import numpy as np
+from netCDF4 import Dataset
 from pyspectral.utils import convert2hdf5 as tohdf5
 from pyspectral.raw_reader import InstrumentRSR
 
 LOG = logging.getLogger(__name__)
 
-FCI_BAND_NAMES = ['ch1', 'ch2', 'ch3', 'ch4', 'ch5', 'ch6', 'ch7', 'ch8', 'ch9', 'ch10',
-                  'ch11', 'ch12', 'ch13', 'ch14', 'ch15', 'ch16']
+FCI_BAND_NAMES = ['VIS0.4', 'VIS0.5', 'VIS0.6_HR', 'VIS0.8', 'VIS0.9', 'NIR1.3',
```
Comment (sfinkens) [inline rsr_convert_scripts/fci_rsr.py:36]: Would it be possible to use `pyspectral.bandnames.BANDNAMES['fci'].values()` here? That would a void duplicating the list.

Comment (adybbroe) [reply rsr_convert_scripts/fci_rsr.py]: Indeed yes, thanks. Much better.

**Review by sfinkens — COMMENTED** [review]
Looks good to me! There are a couple of test misses, but I know too little of pyspectral to judge if that's ok ;)

**Pull request merged**